lib/conjur/api/variables.rb

```diff
@@ -105,6 +105,17 @@ def variable_values(varlist)
       end
     end
 
+    # Fetch all visible variables that expire within the given
+    # interval (expressed as an ISO8601 duration). If no duration is
+    # specifed, all variables that are set to expire will be returned.
+    #
+    # param [String] duration ISO8601 duration 
```

Comment:
Since this is a Ruby API I think it should take a `Range`-ish of `Time`-ishs, instead of exposing the low-level API implementation detail.